python/ray/tune/tests/test_commands.py

```diff
@@ -33,7 +33,7 @@ def __exit__(self, *args):
 
 @pytest.fixture
 def start_ray():
-    ray.init()
+    ray.init(log_to_driver=False)
```

Comment:
You may want to make this change only for `test_ls` since that test captures stdout/stderr. Having the stdout/stderr could make tests easier to debug and also will more accurately simulate real usage.

Reply:
This file is actually mainly for testing the stdout of the CLI (2 of the 3 tests capture output, and `commands.py` is a file for CLI method calls), so I figured it'd make much more sense turning off output logging.